sap-ha-support/xml/art-sap-ha-support.xml

```diff
@@ -0,0 +1,826 @@
+    public cloud vendor are defined in the following &sap; Notes:
+   </para>
+   <informaltable>
+    <tgroup cols="2">
+     <thead>
+      <row>
+       <entry>Company</entry>
+       <entry>&sap; Note</entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>&suse;</entry>
+       <entry><link xlink:href="https://launchpad.support.sap.com/#/notes/1763512">
+       &sap; Note 1763512</link>: Support details for &suse; Linux Enterprise for
+       &sap; Applications - HA Solution for &netweaver; and &sap; &s4hana;</entry>
+      </row>
+      <row>
+       <entry>&ms; Azure</entry>
+       <entry><link xlink:href="https://launchpad.support.sap.com/#/notes/2513384">
+       &sap; Note 2513384</link>: &s4s; on Azure</entry>
+      </row>
+      <row>
+       <entry>&aws;</entry>
+       <entry><link xlink:href="https://launchpad.support.sap.com/#/notes/1656099">
+       &sap; Note 1656099</link>: &sap; Applications on &aws;: Supported DB/OS and &aws; EC2 products</entry>
+      </row>
+      <row>
+       <entry>&gcloud;</entry>
+       <entry><link xlink:href="https://launchpad.support.sap.com/#/notes/2456432">
+       &sap; Note 2456432</link>: &sap; Applications on &gcloud;: Supported Products
+       and GCP VM types</entry>
+      </row>
+     </tbody>
+    </tgroup>
+   </informaltable>
+   <para>
+    For more details, see the &suse; knowledgebase article
+    <link xlink:href="https://www.suse.com/support/kb/doc/?id=000019000">
+    SLES for &sap; - How To Engage &sap; and &suse; to address Product Issues</link>.
+   </para>
+   <note>
+    <para>
+     If necessary, all documented &suse; &ha; scenarios can run the &sap; workload while the
+     &ha; cluster is temporarily disabled.
+    </para>
+   </note>
+  </sect2>
+
+  <sect2  xml:id="sap-ha-support-software">
+   <title>Supported &suse; software versions</title>
+   <para>
+    The general &suse; software lifecycle applies for the described scenarios.
+    See the &suse; lifecycle page for details:
+    <link xlink:href="https://www.suse.com/lifecycle/#suse-linux-enterprise-server-for-sap-applications-15"/>.
+   </para>
+   <para>
+    Usually all mentioned &ha; scenarios are supported on all currently supported service packs of
+    &productname; 15. Exceptions are documented in detail in the setup guides and are listed below:
+   </para>
+   <variablelist>
+    <varlistentry>
+     <term>&hanasr; scale-up - cost optimized scenario</term>
+     <listitem>
+      <para>
+       Supported starting with 15 SP2.
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+   <note>
+    <para>
+     This document applies to &productname; 15. Version 12 SP5 is still supported but not covered by
+     this document, as &suse; strongly recommends using version 15 for new installations.
+     If you need more details, the individual guides cover version 12 as well.
```

Comment:
Let's replace with "If you need details about version 12, check the individual guides available at https://documentation.suse.com/sbp/sap/". 

Background : Some of the linked guides do not covers version 12.